satpy/composites/__init__.py

```diff
@@ -551,69 +551,79 @@ def _insert_palette_colors(channels, palette):
 class DayNightCompositor(GenericCompositor):
     """A compositor that blends a day data with night data."""
 
-    def __init__(self, name, lim_low=85., lim_high=88., **kwargs):
+    def __init__(self, name, lim_low=85., lim_high=88., day_night="day_only", **kwargs):
```

Comment:
Shouldn't the default be `day_night`? And could you mention that it is the default in the docstring below. Something simple like `(default)` next to the name.

Reply:
Yep.